drivers/Makefile.dep

```diff
@@ -100,6 +100,12 @@ ifneq (,$(filter ds1307,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter ds3234,$(USEMODULE)))
+  USEMODULE += ds323x
+  USEMODULE += ds323x_ds3234
```

Comment:
This name seems a bit weird to me. Is it even generated or used somewhere?

Reply:
It's just to get the right file compiled without special tricks to get it recognized as a file from the ds323x directory